markdown_lab/core/scraper.py

```
@@ -253,34 +136,42 @@
                 return cached_content
         return None
 
-    def _start_performance_monitoring(self):
+    def _start_performance_monitoring(self, psutil_available: bool):
         """Start monitoring performance metrics."""
         import time
```

Ruff (0.8.2)

Redefinition of unused `time` from line 10

Remove definition: `time`

(F811)